app/models/service_ansible_playbook.rb

```diff
@@ -3,7 +3,11 @@ class ServiceAnsiblePlaybook < ServiceGeneric
 
   # A chance for taking options from automate script to override options from a service dialog
   def preprocess(action, add_options = {})
-    _log.info("Override with new options: #{add_options}") unless add_options.blank?
+    unless add_options.blank?
+      _log.info("Override with new options:")
+      $log.log_hashes(add_options)
```

Comment:
@bdunne @bzwei 
```_log``` would print out the calling method name as the log prefix while ```$log``` does not. 
Do we really want to print the log prefix for each line of the hash?